src/main/java/no/unit/nva/model/contexttypes/Report.java

```diff
@@ -10,66 +11,39 @@
 
 public class Report extends Book implements SerialPublication, BasicContext {
 
-    private String printIssn;
-    private String onlineIssn;
-
-    public Report() {
-        super();
+    private final String printIssn;
+    private final String onlineIssn;
+
+    public Report(@JsonProperty("series") BookSeries series,
```

Comment:
missing @JsonCreator. Are we sure that  constructors without @JsonCreator  are used when serializing?

Reply:
Same comment here about the `@JsonProperty` values. They should be hardcoded to make the changes easier. Or we can drop the constructors and use the default constructors with getters an  setters.

Reply:
We should have a discussion about this.